### PR TITLE
Speed up certain tests, more reliably set up initial conditions

### PR DESCRIPTION
**Pull request by BlueWinds**
This PR contains a fix for Cy12, and a couple of minor improvements.

- Only logs 20 items instead of 100. I don't see any harm in this reduction, and the benefit is that we run the test a few seconds faster. Waiting for 100 was annoying.
- Cuts out some commands, increasing the 'best practiceness' of this particular example (and, entirely coincidentally, generating a shorter command log and speeding up the test).
- `examples/blogs__vue-vuex-rest/data.json` now properly gets reset when starting the server, and is no longer checked into git. This should mean the example more reliably starts in the same condition every time.
- Also fixes two more examples of invoking async functions, switching them over to `.then()`.

### CODE REVIEW

File: examples/blogs__vue-vuex-rest/data.json [hunk 1]
```diff
@@ -1,3 +1,9 @@
 {
-  "todos": []
+  "todos": [
```
[the remark this answers is not in the document]
Comment (BlueWinds) [reply examples/blogs__vue-vuex-rest/data.json]: Oh, whoops, I had it removed in a local commit which hadn't been pushed yet.

**Comment by BlueWinds** [comment]
This PR fixes one failure; I'd like to merge it in, and then address further failures in future PRs.

**Comment by emilyrohrbough** [comment]
@BlueWinds Do you know the source of the test failures in this PR?

**Comment by BlueWinds** [comment]
@emilyrohrbough - I don't yet, haven't looked into it. My suspicion is that it is related to Cypress 12 in some way, just haven't spent time on it yet. Planning on getting to that tomorrow.

That job is failing in 12.0.0 already, though not quite as verbosely.

**Review by emilyrohrbough — APPROVED** [review]

**Pull request merged**